recipes/libfranka/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+set -exo pipefail
+
+cmake -B build -DBUILD_TESTS=OFF -DBUILD_EXAMPLES=OFF -DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_INSTALL_LIBDIR=lib $SRC_DIR -DCMAKE_POLICY_VERSION_MINIMUM=3.5
```

Comment:
You should use `$CMAKE_ARGS` here, that contains for example `-DCMAKE_INSTALL_PREFIX=$PREFIX -DCMAKE_INSTALL_LIBDIR=lib`.